giraffe/src/components/SimpleTable/PagedTable.tsx

```diff
@@ -230,18 +241,19 @@ const subsetResult = (
 }
 
 interface Props {
-  properties: SimpleTableViewProperties
+  showAll: boolean
   result: FluxResult['parsed']
 }
 
 const INITIAL_HEADER_HEIGHT = 0
 const INITIAL_HEIGHT = 0
-const INITIAL_ROW_HEIGHT = 10 // must be greater than 0
-const PagedTable: FC<Props> = ({result, properties}) => {
+const INITIAL_ROW_HEIGHT = 0
```

Comment:
The initial row height is indeed 0. The calculations that estimate the number of rows should handle its own logic in not using a 0 height to cause an infinite number of possible rows (and thus running through the entire data set). Thus, what we're doing here is moving the "must be greater than 0" logic to the correct place rather than arbitrarily putting it into this constant.